crm_lead_attach_related_saleorder_report/models/crm_lead.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 Valentin Vinagre <valentin.vinagre@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class CrmLead(models.Model):
+    _inherit = "crm.lead"
+
+    def action_send_documents_sale(self):
+        """
+        This function opens a window to compose an email.
+        """
+        self.ensure_one()
+        ir_model_data = self.env["ir.model.data"]
+        try:
+            compose_form_id = ir_model_data.get_object_reference(
```

Comment:
Where is the get_object_reference method declared? I've checked the ir_model.py file in base module and it is not there. Am I missing something?